content/kotsadm/gitops/multi-app-workflows.md

```diff
@@ -6,46 +6,42 @@ weight: 2
 draft: false
 ---
 
-While the Admin Console is initially configured to receive updates, show the changes and deploy them to the cluster, this process can be changed and converted to use a GitOps workflow instead. 
+While the Admin Console is initially configured to receive updates, show the changes and deploy them to the cluster, this process can be changed and converted to use a GitOps workflow instead.
 When using a GitOps workflow, changes from the Admin Console (config changes, upstream updates, license updates) will be pushed to a private Git repository, where an existing CI/CD process can execute to deliver the manifests to the cluster.
 
-To begin migrating to a GitOps deployment workflow, click the GitOps link at the top of the Admin Console, then click on "Get started".
+To begin migrating to a GitOps deployment workflow:
 
-![GitOps](/images/gitops.png)
+1. Click the GitOps link at the top of the Admin Console, then click on "Get started".
 
-Continuing, choose the Git provider and hostname (if applicable), then click on "Finish GitOps setup".
+    The GitOps Provider page opens.
 
-![GitOps Provider](/images/gitops-provider-multi-app.png)
+1. Choose the Git provider and hostname (if applicable), then click on "Finish GitOps setup".
 
-A screen displaying all of your apps with the status of GitOps integration for each app will show up next. 
-Click the "Enable" button next to the app you want to enable GitOps for.
+    A screen displaying all of your apps with the status of GitOps integration for each app will show up next.
 
-![GitOps Provider](/images/gitops-apps.png)
+1. Click the "Enable" button next to the app you want to enable GitOps for.
 
-You'll be taken to the GitOps settings page for that app. Enter the repo details, and optionally branch and path to commit to in the repo.
+    ![GitOps Provider](/images/gitops-apps.png)
 
-Finally, select the action to take when there's an update. 
-The Admin Console can either create a new commit to the branch specified, or it can create a Pull/Merge Request (if supported by the provider). 
-On this screen, there's an option to choose what type of asset to deliver to the git repo.
+    You'll be taken to the GitOps settings page for that app.
```

Comment:
Same comment as above, I would move this up to the step and not do a line break so it isn't lost amid the graphic.

Reply:
I refer keeping the results step as a separate paragraph because it's easier to scan. But, I can go either way and we can let Paige weigh in too. I guess we should consult the Chicago Manual of Style when we do the rewrites for the new docs site. But for now, we can take your changes as is.